exercise3.R

```diff
@@ -0,0 +1,30 @@
+#Install tidyverse packages for this activity
+
+install.packages("tidyverse")
+install.packages("ggthemes")
```

Comment:
I just told you this yesterday, we should not encode in a script the `install.packages()` command.